sw/host/opentitanlib/bindgen/BUILD

```diff
@@ -254,7 +254,6 @@ rust_bindgen_library(
         "--allowlist-function=status_err",
         "--allowlist-function=status_ok",
         "--generate-inline-functions",
-        "--with-derive-custom=ot_status_create_record=zerocopy::FromZeroes",
```

Comment:
Why don’t we need this anymore?

Reply:
I mentioned it in the PR body and commit message:
> `FromBytes` now imply `FromZeroes` and doesn't need explicit derive